### PR TITLE
ASU-703 Add apartment presentation link

### PR DESCRIPTION
**Pull request by genie9**
Add Helsinki appartment office presentation link to apartment at the end
of Oikotie and Etuovi apartment description text.

### CODE REVIEW

File: connections/oikotie/oikotie_mapper.py [hunk 1]
```diff
@@ -367,6 +367,20 @@ def map_new_development_status(
...
+    Fetch link to apartment presentation and add it to the end of project description
+    """
+    main_text = getattr(elastic_apartment, "project_description", None)
+    link = getattr(elastic_apartment, "url", None)
+    link_message = "Tarkempi esittely Helsingin kaupungin asuntotuotannon sivulla"
+
+    if main_text and link:
+        return f"{main_text}\n{link_message} {link}"
+    else:
+        return main_text
```
Comment (tomijarvi) [inline connections/oikotie/oikotie_mapper.py:381]: Should it send a link if the main text hasn't been set? Ex. like this?
```
link_text =  f"{link_message} {link}" if link else None

if main_text or link_text:
    return "\n".join(filter(None, [main_text, link_text]))
return None

```

The same applies also to the Etuovi side.

**Review by tomijarvi — COMMENTED** [review]
Nice 👍  Check my comments.

File: connections/etuovi/etuovi_mapper.py [hunk 2]
```diff
@@ -239,6 +239,20 @@ def map_scontacts(elastic_apartment: Apartment) -> Optional[List[Scontact]]:
     return None
 
 
+def form_presentation(elastic_apartment):
+    """
+    Fetch link to apartment presentation and add it to the end of project description
+    """
+    main_text = getattr(elastic_apartment, "project_description", None)
+    link = getattr(elastic_apartment, "url", None)
+    link_message = "Tarkempi esittely Helsingin kaupungin asuntotuotannon sivulla"
```
[the remark this answers is not in the document]
Comment (genie9) [reply connections/etuovi/etuovi_mapper.py]: no, took it away

File: connections/oikotie/oikotie_mapper.py (hunk 1 above)
Comment (genie9) [reply connections/oikotie/oikotie_mapper.py]: Not sure... Should this be agreed with HKI that description can be empty?

Comment (tomijarvi) [reply connections/oikotie/oikotie_mapper.py]: Previously the description was nullable in the code, so it should be already agreed. But it is good to make sure that if no description is set, then we will still put the link to go if that exists.

**Review by tomijarvi — APPROVED** [review]

**Pull request merged**